pkg/olm/clusterserviceversion.go

```diff
@@ -4,13 +4,13 @@ import (
 	"context"
 	"fmt"
 
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
-
 	"github.com/golang/glog"
 	"github.com/openshift-kni/eco-goinfra/pkg/clients"
 	"github.com/openshift-kni/eco-goinfra/pkg/msg"
-	oplmV1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	oplmV1alpha1 "github.com/openshift-kni/eco-goinfra/pkg/schemes/olm/operators/v1alpha1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtimeClient "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Since this PR uses testSchemes and the runtimeClient, should it also include switching `ClusterServiceVersionBuilder.apiClient` to be `runtimeClient.Client`? Or is that in a future PR?

Reply:
Due to the fact that I need to switch the rest of the objects under olm operator I decided to implement it later. I don's want to have huge PRs.